coords_systems.md

```diff
@@ -0,0 +1,85 @@
+Coordinate Systems in L5Kit
+===
+
+# Introduction
+One essential feature of L5Kit is converting raw data into multi-channel images. We refer to this process as
+**rasterisation**. These raw data can come from different sources, such as:
+- `.zarr` datasets, which include information about the AV and other agents;
+- static aerial images;
+- dynamic semantic maps, which include lane topologies, crosswalks, etc..
+
+Clearly, these different sources can have different coordinates systems, which need to be converted to a single common system
+before we can use them together to get our final multi-channel image. 
+L5Kit performs this operation smoothly under the hood, but you may want to know more details about these different systems
+if you're trying a more experimental workflow.
+
+# Coordinates Systems
+
+## World Coordinate System
+We refer to the coordinate system in the `.zarr` dataset as **world** or **pose** (TODO chose one). This is shared by *all* our zarrs in a dataset
```

Comment:
Let's fix this TODO? I do not have enough  experience in these to feel like making a decision though - from how it sounds, I like world though.

Reply:
world +1

Reply:
we set our mind on world :)